newton/_src/sim/collide_unified.py

```
@@ -329,68 +328,67 @@ class CollisionPipelineUnified:
         self,
-        shape_count: int,
-        particle_count: int,
+        model: Model,
+        *,
         reduce_contacts: bool = True,
-        shape_pairs_filtered: wp.array(dtype=wp.vec2i) | None = None,
         rigid_contact_max: int | None = None,
         soft_contact_max: int | None = None,
         soft_contact_margin: float = 0.01,
-        edge_sdf_iter: int = 10,
         iterate_mesh_vertices: bool = True,
```

Ruff (0.14.14)

[warning] Unused method argument: `iterate_mesh_vertices`

(ARG002)